homeassistant/components/reolink/const.py

```diff
@@ -7,3 +7,5 @@
 
 DEFAULT_PROTOCOL = "rtsp"
 DEFAULT_TIMEOUT = 60
+
+SUBSCRIPTION_RENEW_THRESHOLD = 300
```

Comment:
If the constant is only used within a single module please define it within that module. It makes it easier for readers to read the module.

Reply:
I moved as much constants as possible